lib/mayaUsd/render/vp2RenderDelegate/render_delegate.cpp

```diff
@@ -289,12 +349,14 @@ namespace
         bool                    _isInitialized { false };  //!< Whether the shader cache is initialized
 
         //! Shader registry used by fallback shaders
-        MShaderMap              _fallbackShaders[static_cast<size_t>(FallbackShaderType::kCount)];
+        MShaderMap              _fallbackShaders[FallbackShaderTypeCount];
         MShaderMap              _3dSolidShaders;
 
-        MHWRender::MShaderInstance*  _fallbackCPVShader { nullptr }; //!< Fallback shader with CPV support
-        MHWRender::MShaderInstance*  _3dFatPointShader { nullptr };  //!< 3d shader for points
-        MHWRender::MShaderInstance*  _3dCPVSolidShader { nullptr };  //!< 3d CPV solid-color shader
+        //!< Fallback shaders with CPV support
+        MHWRender::MShaderInstance* _fallbackCPVShaders[FallbackShaderTypeCount];
```

Comment:
Can you initialize the array elements to nullptr?

Reply:
Good point. Thanks.